external/substrate/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "substrate-minimal-runtime"
+version = "0.1.0"
+description = "Minimal Substrate runtime for testing compatibility"
+publish = false
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+frame-support = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+risc0-zkvm = { workspace = true, default-features = false }
+
+[build-dependencies]
+substrate-wasm-builder = { version = "5.0.0-dev", git = "https://github.com/paritytech/substrate.git", optional = true , branch = "polkadot-v1.0.0" }
```

Comment:
Looks like this has been archived. You mentioned this repo on discord - https://github.com/paritytech/polkadot-sdk/tree/master/substrate/utils/wasm-builder

Could you use this one instead since is more frequently maintained?

Reply:
Ah, yeap; good catch. That's the new monorepo they just switched everything over to